### ISSUE TITLE
How to call mongodb javascript thorugh Kundera

### ISSUE THREAD

**Issue by sbheda** [Impetus/kundera#248]
I have one java script  in mongodb like 
db.system.js.save(
                   { _id: "echoFunction",
                     value : function(x) { return x; }
                   }
 )
how can I call this java script using kundera ?

**Comment by mevivs** [comment]
I have added a fix to enable this, here is commit revision number:
6654e0b9ce8b511f0177f4fd23a5cb37f01dd259

Please have a look at:
 https://github.com/impetus-opensource/Kundera/blob/trunk/kundera-mongo/src/test/java/com/impetus/client/crud/PersonMongoTest.java

in onExecuteScript() method for sample junits.

given below is sample code snippet:

```
  private void onExecuteScript()
    {
        Map<String, Client<Query>> clients = (Map<String, Client<Query>>) em.getDelegate();
        Client client = clients.get(_PU);

        String jScript = "db.system.js.save({ _id: \"echoFunction\",value : function(x) { return x; }})";
        Object result = ((MongoDBClient)client).executeScript(jScript);
        Assert.assertNull(result);
        String findOneJScript = "db.PERSON.findOne()";
        result = ((MongoDBClient)client).executeScript(findOneJScript);
        Assert.assertNotNull(result);
    }


```

**Comment by mevivs** [comment]
Fixed and released with Kundera 2.5. Please verify.

-Vivek

**Comment by mevivs** [comment]
Closing. Already released with 2.5
